test/system/system.test.ts

```diff
@@ -411,6 +421,37 @@ test("inspect redis:3.2.11-alpine", (t) => {
         },
         "deps",
       );
+
+      const manifestFilesResult = res.scannedProjects.find((res) => {
+        if (!("scanType" in res)) {
+          return false;
+        }
+        const scannedProjectExtended = res as ScannedProjectExtended;
+        if (scannedProjectExtended.scanType === ScanType.ManifestFiles) {
```

Comment:
Just a simple return statement is fine here...
```suggestion
        return scannedProjectExtended.scanType === ScanType.ManifestFiles
```